<ui version="4.0" >
 <class>MMCSettingFormClass</class>
 <widget class="QDialog" name="MMCSettingFormClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MMCSettingForm</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QGroupBox" name="groupBox" >
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QCheckBox" name="chShowOKMessage" >
        <property name="text" >
         <string>良品メッセージウィンドウを表示する</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="chShowNGArea" >
        <property name="text" >
         <string>NGと判定した個片名称を履歴リストに表示する</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QCheckBox" name="chNotShowEID" >
        <property name="text" >
         <string>検査ID（ソフト側）を履歴リストに表示しない</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QCheckBox" name="chOKDoubleDone" >
        <property name="text" >
         <string>最終確認画面をOKボタン２回押しで確定する</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QCheckBox" name="chNGListControl" >
        <property name="text" >
         <string>NGリストのマウス操作を制限する</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QGroupBox" name="gbPickup" >
        <property name="title" >
         <string>抜き取り設定</string>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <widget class="QGroupBox" name="groupBox_2" >
           <property name="title" >
            <string>品種A</string>
           </property>
           <layout class="QGridLayout" >
            <item row="0" column="0" >
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>NG個片数がこの値以上の場合、メッセージを出す</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QSpinBox" name="sbPickupNGAreaNumbA" >
              <property name="value" >
               <number>5</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QGroupBox" name="groupBox_3" >
           <property name="title" >
            <string>品種B</string>
           </property>
           <layout class="QGridLayout" >
            <item row="0" column="0" >
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>NG個片数がこの値以上の場合、メッセージを出す</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QSpinBox" name="sbPickupNGAreaNumbB" >
              <property name="value" >
               <number>3</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QGroupBox" name="groupBox_4" >
           <property name="title" >
            <string>品種C</string>
           </property>
           <layout class="QGridLayout" >
            <item row="0" column="0" >
             <widget class="QLabel" name="label_3" >
              <property name="text" >
               <string>NG個片数がこの値以上の場合、メッセージを出す</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QSpinBox" name="sbPickupNGAreaNumbC" >
              <property name="value" >
               <number>4</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>上流側のログ保存先フォルダを指定（下流側設定）</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" >
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QLineEdit" name="leLogSaveDirectory" >
             <property name="readOnly" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbLogSaveDirectory" >
             <property name="minimumSize" >
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="text" >
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QLabel" name="label_9" >
     <property name="text" >
      <string>ログデータの出力先を指定</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLineEdit" name="leLogOutputDirectory" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbLogOutputDirectory" >
       <property name="minimumSize" >
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <widget class="QPushButton" name="pbOK" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>41</height>
      </size>
     </property>
     <property name="text" >
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>171</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2" >
    <widget class="QPushButton" name="pbCancel" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>41</height>
      </size>
     </property>
     <property name="text" >
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
